content/src/roadmaps/steps.json

```diff
@@ -1,5 +1,12 @@
 {
   "steps": [
+    {
+      "section": "DOMESTIC_EMPLOYER_SECTION",
+      "stepNumber": 1,
+      "name": "Register as an Employer",
+      "timeEstimate": "",
+      "description": ""
+    },
     {
       "section": "PLAN",
       "stepNumber": 1,
```

Comment:
Not sure why but it also appears to be starting with the area collapsed 
<img width="1275" alt="image" src="https://github.com/user-attachments/assets/6c6dbd10-0d4c-437c-b525-a69be011574c">
Probably not what we want, I don't see something obvious as to why but probably worth looking into a bit deeper

Reply:
That's a great callout that I did not catch! Thank you!